### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by mathieu-fanduel**

### CODE REVIEW

**Comment by nightm4re94** [comment]
Hi, just checking in if this PR is still in draft state or ready for review?
cheers!

**Comment by mathieu-fanduel** [comment]
Actually it could be reviewed , these are some hopefully minimal changes ( fixing typo ) to get my teeth in the code base and review process

**Comment by nightm4re94** [comment]
Hi, sorry this is taking longer than usual. We're on a sort of break from LITIENGINE due to heavy workload currently.
What I can say however is that we would prefer PullRequests that are separated into different topics, for example:
- one for fixing typos
- one for proposing Entity opacity
- one for Java 17 refactorings
This speeds up reviews and lets us merge parts even if other parts need revisions.

**Comment by mathieu-fanduel** [comment]
Cool I will do that

**Comment by nightm4re94** [comment]
Hi there, just checking in whether you're still interested in splitting this PR up into three separate ones. That would facilitate the review greatly.

File: core/src/main/java/de/gurkenlabs/litiengine/Game.java [hunk 1]
```diff
@@ -446,7 +446,7 @@ public static TweenEngine tweens() {
    * </ul>
    *
    * @param args
-   *          The arguments passed to the programs entry point.
+   *          The arguments passed to the programs' entry point.
```
Comment (nightm4re94) [inline core/src/main/java/de/gurkenlabs/litiengine/Game.java:449]:
```suggestion
   *          The arguments passed to the program's entry point.
```

File: core/src/main/java/de/gurkenlabs/litiengine/attributes/AttributeModifier.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package de.gurkenlabs.litiengine.attributes;
 
 /**
- * An attribute modifier allows to modify attributes by the specified Modification and modify value.
+ * An attribute modifier than allows modifying attributes by the specified Modification and modify value.
```
Comment (nightm4re94) [inline core/src/main/java/de/gurkenlabs/litiengine/attributes/AttributeModifier.java:4]:
```suggestion
 * An attribute modifier that allows modifying attributes by the specified Modification and modify value.
```

File: core/src/main/java/de/gurkenlabs/litiengine/entities/behavior/AStarPathFinder.java [hunk 3]
```diff
@@ -1,16 +1,16 @@
 package de.gurkenlabs.litiengine.entities.behavior;
 
-import de.gurkenlabs.litiengine.entities.IMobileEntity;
-import de.gurkenlabs.litiengine.environment.tilemap.IMap;
-import java.awt.Dimension;
-import java.awt.Point;
+import java.awt.*;
```
Comment (nightm4re94) [inline core/src/main/java/de/gurkenlabs/litiengine/entities/behavior/AStarPathFinder.java:3]: Please don't use wildcard imports.

File: core/src/main/java/de/gurkenlabs/litiengine/entities/behavior/AStarPathFinder.java [hunk 4]
```diff
@@ -42,6 +42,9 @@ public Path findPath(final IMobileEntity entity, final Point2D target) {
 
     final AStarNode startNode = this.getGrid().getNode(startLocation);
     AStarNode targetNode = this.getGrid().getNode(target);
+    if ( targetNode == null ) {
```
Comment (nightm4re94) [inline core/src/main/java/de/gurkenlabs/litiengine/entities/behavior/AStarPathFinder.java:45]: You could combine the two `if`-conditions.

File: core/src/main/java/de/gurkenlabs/litiengine/graphics/RenderEngine.java [hunk 5]
```diff
@@ -1,5 +1,16 @@
 package de.gurkenlabs.litiengine.graphics;
 
+import java.awt.*;
```
Comment (nightm4re94) [inline core/src/main/java/de/gurkenlabs/litiengine/graphics/RenderEngine.java:3]: No wildcard imports, please.

File: core/src/main/java/de/gurkenlabs/litiengine/entities/Entity.java [hunk 6]
```diff
@@ -68,6 +68,8 @@ public abstract class Entity implements IEntity, EntityRenderListener, Tweenable
   @TmxProperty(name = MapObjectProperty.RENDERTYPE)
   private RenderType renderType;
 
+  private float opacity = 1f;
```
Comment (nightm4re94) [inline core/src/main/java/de/gurkenlabs/litiengine/entities/Entity.java:71]: While the idea of setting the drawing opacity is nice, I don't approve how it's implemented:
We should keep rendering-specific stuff separate from the Entity logic. 
The preferred way to achieve this visual effect would be to apply an ImageEffect in the Entity's AnimationController.

Please remove all Entity opacity related code from this PR.

**Review by nightm4re94 — CHANGES_REQUESTED** [review]
Thanks a lot for this PR and please excuse me for taking a while with this review.
Two remarks:
* please remove all entity opacity related code
* please make sure to rebase the branch onto master and run the spotlessApply Gradle task to format everything according to our style guide.

Thanks again!

File: core/src/main/java/de/gurkenlabs/litiengine/entities/Entity.java (hunk 6 above)
Comment (nightm4re94) [reply core/src/main/java/de/gurkenlabs/litiengine/entities/Entity.java:71]:
```suggestion
```

**Comment by nightm4re94** [comment]
Closing this due to inactivity. Thank you and feel free to resume at a later point :)

**Pull request closed without merging**